ami/ml/post_processing/tracking_task.py

```
@@ -0,0 +1,485 @@
+                    d.occurrence = keeper
+                    d.save()
+
+            # Delete now-empty sibling occurrences. v1's fresh-event invariant guarantees
+            # these have no Identifications attached (nothing has been ratified yet), so
+            # CASCADE on Identification.occurrence is harmless. v2 must instead reassign
+            # Identification.occurrence to the keeper before deleting.
+            for occ_id in old_occ_ids - {keeper.pk}:
+                try:
+                    Occurrence.objects.filter(id=occ_id).delete()
+                    merged += 1
+                except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)